webpack.config.js

```diff
@@ -109,10 +109,30 @@ module.exports = {
             loader: '@svgr/webpack',
             options: {
               svgoConfig: {
-                plugins: {
-                  // SVGR removes viewbox by default. Disable this behavior.
-                  removeViewBox: false
-                }
+                plugins: [
+                  {
+                    // cf. https://github.com/svg/svgo/releases/tag/v2.4.0
+                    name: 'preset-default',
+                    params: {
+                      overrides: {
+                        // don't minify "id"s (i.e. turn randomly-generated unique ids into "a", "b", ...)
+                        // https://github.com/svg/svgo/blob/master/plugins/cleanupIDs.js
+                        cleanupIDs: { minify: false },
+                        // leave <line>s, <rect>s and <circle>s alone
+                        // https://github.com/svg/svgo/blob/master/plugins/convertShapeToPath.js
+                        convertShapeToPath: false,
+                        // leave "class"es and "id"s alone
+                        // https://github.com/svg/svgo/blob/master/plugins/prefixIds.js
+                        prefixIds: false,
+                        // leave "stroke"s and "fill"s alone
+                        // https://github.com/svg/svgo/blob/master/plugins/removeUnknownsAndDefaults.js
+                        removeUnknownsAndDefaults: { defaultAttrs: false },
+                        // leave viewBox alone
+                        removeViewBox: false
```

Comment:
If it's really true that svgo removes the `viewBox` by default, then we should probably add this to the default configuration for CLUE, DESE, starter-projects, etc. Odd that I've never noticed that. 🤔

Reply:
The main splash screen in TE is broken without that rule:
<img width="765" alt="Screenshot 2022-01-27 at 17 48 58" src="https://user-images.githubusercontent.com/767857/151473016-42a5ce7c-fb51-4e96-acc0-7bfaf21d9f6d.png">
It was also visible when I inspected this SVG and tried to re-add viewBox by hand.

Reply:
>The splash screen in TE is broken without that rule

That also suggests that it's a new default optimization in v6 given that you weren't experiencing it before. Given that, I'm surprised we haven't seen more breakage. 🤔

Reply:
Actually, it was already in v5 - you can take a look at the diff here. The old SVGR version was v5, and the old config was:
```
plugins: {
  // SVGR removes viewbox by default. Disable this behavior.
  removeViewBox: false
}
```
I looked at the starter-projects, as I couldn't disable `removeViewBox` using weird SVGR v6 options / plugins syntax. :wink: And then transferred other options too.

Re viewBox itself - maybe we're just lucky. Most of the SVG files should work fine with viewBox. Even when I look at splash screen here, I don't understand why it needs viewBox to work fine (but I confirmed that using devtools). Or... maybe SVGr does some magic transformations of all the SVG element coordinates so it can remove viewBox (I wouldn't be surprised look how many attributes it modifies).

Reply:
Makes sense -- all the more reason to make it part of our default configuration. As far as I'm concerned we should disable any optimization that can potentially break things.